doc/changelog.rst

```diff
@@ -1,14 +1,27 @@
 Changelog
 =========
 
-Changes in Version 3.12.0
+Changes in Version 3.11.1
 -------------------------
 
-Version 3.12 adds support for Python 3.9 and includes a number of bug fixes.
+Version 3.11.1 adds support for Python 3.9 and includes a number of bugfixes.
 Highlights include:
 
 - Support for Python 3.9.
-- New method :class:`bson.json_util.JSONOptions.with_options`.
+- Initial support for Azure and GCP KMS providers for client side field level
+  encryption is in beta. See the docstring for
+  :class:`~pymongo.mongo_client.MongoClient`,
+  :class:`~pymongo.encryption_options.AutoEncryptionOpts`,
+  and :mod:`~pymongo.encryption`. **Note: Backwards-breaking changes may be
+  made before the final release.**
+- Fixed a bug where the :class:`bson.json_util.JSONOptions` API did not match
+  the class:`bson.codec_options.CodecOptions` API due to the absence of
+  a :meth:`bson.json_util.JSONOptions.with_options` method. This method has now
+  been added.
+- Fixed a bug wherein PyMongo did not always mark an implicit session
+  "dirty" after encountering a network error.
```

Comment:
"not always mark an implicit session "dirty" after" -> "did not always discard an implicit session after ..."